src/features/books/booksSlice.js

```diff
@@ -0,0 +1,26 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const booksSlice = createSlice({
```

Comment:
- [ ] According to project requirements, the structure of the files should follow the `ducks pattern`, please kindly move booksSlice.js to `/src/redux/books/booksSlice.js`